hack/test-integration.sh

```diff
@@ -13,14 +13,48 @@ cd "${OS_ROOT}"
 
```

Comment:
why wouldn't we just set `ETCD_PORT=44001` when we invoked the integration tests somewhere we wanted them parallelized?

probably need ETCD_PEER_PORT as well, but still...